#! /bin/sh /usr/share/dpatch/dpatch-run
## 00_hardy_menu_patch.dpatch by m.eik michalke <m@reaktanz.de>
##
## All lines beginning with `## DP:' are a description of the patch.
## DP: A patch for the menu entry in Ubuntu 8.04

@DPATCH@
diff -urNad rkward-0.5.0d-rc1~/rkward/rkward.desktop rkward-0.5.0c/rkward/rkward.desktop
--- rkward-0.5.0d-rc1~/rkward/rkward.desktop	2009-03-30 11:40:58.000000000 +0200
+++ rkward-0.5.0d-rc1/rkward/rkward.desktop	2009-03-30 23:32:09.000000000 +0200
@@ -5,7 +5,7 @@
 Comment[pl]=Graficzny interfejs dla projektu R
 Comment[fr]=Interface graphique pour R
 Icon=rkward
-Exec=rkward -caption "%c" %i
+Exec=/usr/lib/kde4/bin/rkward -caption "%c" %i
 Terminal=false
 Type=Application
 DocPath=rkward/index.html
